### PR TITLE
Change home title to all caps

### PR DESCRIPTION
**Pull request by nfuent02**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def display_page(pathname):
         return page2.layout
     else:
         return html.Div(id='main-div', children=[
-            html.H3('Home'),
+            html.H3('HOME'),
```
Comment (eufemianofp) [inline main.py:25]: Todo con mayusculas?

**Review by eufemianofp — APPROVED** [review]
First review

File: main.py (hunk 1 above)
Comment (nfuent02) [reply main.py:25]: obvio

Comment (nfuent02) [reply main.py:25]: mando yo

**Pull request merged**